velox/dwio/common/Options.h

```diff
@@ -617,6 +619,7 @@ struct WriterOptions {
   std::map<std::string, std::string> serdeParameters;
   std::optional<uint8_t> zlibCompressionLevel;
   std::optional<uint8_t> zstdCompressionLevel;
+  ParquetDataPageVersion parquetDataPageVersion;
```

Comment:
Parquet specific options now should go to its own `WriterOptions`, see #10470

Reply:
@Yuhta Yes, thanks. PR was still in draft state as I was working through the rebase. 

Separated out parquet options now, and cleaned it all up - would appreciate if you could take a look again!